.github/workflows/docs.yml

```diff
@@ -17,42 +17,44 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      
+
       - uses: actions/setup-python@v4
         with:
           python-version: '3.x'
 
       - name: Setup doc dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r docs/requirements.txt
-      
+          python -m pip install -r docs/requirements.txt
+          python -m pip install .
```

Comment:
As `snakemake` needs to be imported for the docs to build then need to actually install it, and not import the `snakemake/` directory at the top level as was previously done.